lib/SILGen/SILGenDecl.cpp

```diff
@@ -1522,7 +1522,26 @@ void SILGenFunction::emitStmtCondition(StmtCondition Cond, JumpDest FalseDest,
     }
 
     case StmtConditionElement::CK_HasSymbol: {
-      llvm::report_fatal_error("Can't SILGen #_hasSymbol yet!");
+      auto info = elt.getHasSymbolInfo();
+      if (info->isInvalid()) {
+        SILType i1 = SILType::getBuiltinIntegerType(1, getASTContext());
+        booleanTestValue = B.createIntegerLiteral(loc, i1, 1);
```

Comment:
Does this mean we fall through if we are unable to check availability meaningfully? Is it worth diagnosing here?

Reply:
This is an ill-conceived attempt to handle bad input from Sema. I think I'll change this to assert `! info->isInvalid()`.

Reply:
Makes sense to me!